pkg/api/cmd/volume_ls.go

```diff
@@ -0,0 +1,36 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package cmd
+
+import (
+	"github.com/containerd/nerdctl/pkg/api/types"
+
+	"github.com/spf13/cobra"
```

Comment:
`pkg/...` is expected to be isolated from `spf13/cobra`

Reply:
I cannot really understand what "Create some file in pkg/api/cmd/${cmd}, and move the command entry point in real into this package" means. Would you mind give more specific guidance?

Reply:
https://github.com/containerd/nerdctl/pull/1774/files

@miles170 I have made a example PR, this may help you